data/emulator.js

```diff
@@ -2491,132 +2493,125 @@ class EmulatorJS {
             },
             9: {
                 'value': 'a',
-                'value2': 'BUTTON_3',
-                'keycode': 65
+                'value2': 'BUTTON_3'
             },
             10: {
                 'value': 'q',
-                'value2': 'LEFT_TOP_SHOULDER',
-                'keycode': 81
+                'value2': 'LEFT_TOP_SHOULDER'
             },
             11: {
                 'value': 'e',
-                'value2': 'RIGHT_TOP_SHOULDER',
-                'keycode': 69
+                'value2': 'RIGHT_TOP_SHOULDER'
             },
             12: {
-                'value': 'e',
-                'value2': 'LEFT_BOTTOM_SHOULDER',
-                'keycode': 69
+                'value': 'tab',
+                'value2': 'LEFT_BOTTOM_SHOULDER'
             },
             13: {
-                'value': 'w',
-                'value2': 'RIGHT_BOTTOM_SHOULDER',
-                'keycode': 87
+                'value': 'r',
+                'value2': 'RIGHT_BOTTOM_SHOULDER'
             },
             14: {
+                'value': '',
                 'value2': 'LEFT_STICK',
             },
             15: {
+                'value': '',
                 'value2': 'RIGHT_STICK',
             },
             16: {
                 'value': 'h',
-                'value2': 'LEFT_STICK_X:+1',
-                'keycode': 72
+                'value2': 'LEFT_STICK_X:+1'
             },
             17: {
                 'value': 'f',
-                'value2': 'LEFT_STICK_X:-1',
-                'keycode': 70
+                'value2': 'LEFT_STICK_X:-1'
             },
             18: {
                 'value': 'g',
-                'value2': 'LEFT_STICK_Y:+1',
-                'keycode': 71
+                'value2': 'LEFT_STICK_Y:+1'
             },
             19: {
                 'value': 't',
-                'value2': 'LEFT_STICK_Y:-1',
-                'keycode': 84
+                'value2': 'LEFT_STICK_Y:-1'
             },
             20: {
                 'value': 'l',
-                'value2': 'RIGHT_STICK_X:+1',
-                'keycode': 76
+                'value2': 'RIGHT_STICK_X:+1'
             },
             21: {
                 'value': 'j',
-                'value2': 'RIGHT_STICK_X:-1',
-                'keycode': '74'
+                'value2': 'RIGHT_STICK_X:-1'
             },
             22: {
                 'value': 'k',
-                'value2': 'RIGHT_STICK_Y:+1',
-                'keycode': '75'
+                'value2': 'RIGHT_STICK_Y:+1'
             },
             23: {
                 'value': 'i',
-                'value2': 'RIGHT_STICK_Y:-1',
-                'keycode': '73'
+                'value2': 'RIGHT_STICK_Y:-1'
+            },
+            24: {
+                'value': '1'
+            },
+            25: {
+                'value': '2'
+            },
+            26: {
+                'value': '3'
+            },
+            27: {
```

Comment:
I dont think we should have a mapping for fast foward, slow motion, and rewind. Especially since most keyboards dont really have an "add" or "subtract" button, and space seems like something you would automatically set. I think we should leave this up to the user

Reply:
Done!